debugger_implementation.tex

```diff
@@ -392,6 +392,81 @@ \subsection{Using Program Buffer} \label{deb:mrprogbuf}
 \end{tabulary}
 \medskip
 
+\section{Triggers}
+
+A debugger would use hardware triggers to halt a hart when a certain event
```

Comment:
would -> can

Also, the use of Trigger is not unique to a debugger, so might say "a debugger or more privileged mode can"

Reply:
Oh, I just realized this is in "debugger implementation" section, so maybe not necessary to distinguish there